config/initializers/devise.rb

```diff
@@ -19,7 +19,7 @@
   # note that it will be overwritten if you use your own mailer class
   # with default "from" parameter.
   config.mailer_sender = 'please-change-me-at-config-initializers-devise@example.com'
-  config.omniauth :facebook, "2420959254817666", "4e1eea18dc615cf9408bef7342e532b7", callback_url: "https://fast-spire-95171.herokuapp.com/users/auth/facebook/callback"
+  config.omniauth :facebook, "2420959254817666", "4e1eea18dc615cf9408bef7342e532b7", callback_url: "https://intense-harbor-38641.herokuapp.com/users/auth/facebook/callback"
```

Comment:
it's better to add the private keys in the global environment variables in heroku

![heroku](https://user-images.githubusercontent.com/34820784/68515212-6e6e1900-0280-11ea-8412-8fe5832a1704.png)